### PR TITLE
Adding new Pachyderm example (picking up where #112 left-off

### PR DESCRIPTION
**Pull request by Nick-Harvey**
I spoke with Dan Whitenack to see what needed to be done to get [pull request #112](https://github.com/kubeflow/examples/pull/112) across the line. Dan being the busy guy he is mentioned that he wasn't going to be able to get to it for a little while, and suggested I pick it up. So, here is his exact work (he should get 100% credit for this PR) with a few minor edits from me.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/282)

### CODE REVIEW

**Comment by Nick-Harvey** [comment]
> https://cla.developers.google.com/

I signed it

**Comment by Nick-Harvey** [comment]
Regarding the KubeFlow install section, I went ahead and just removed all the manual steps we had there, and instead just point users to the KubeFlow official getting started page. Doing so will make sure that users are always using the official Kubeflow install instructions, and we won't run the risk of having custom install instructions falling behind the latest official ones. 

As for the TFjob, I'm planning on that as a follow up to this example (est. mid-November). Hopefully, that doesn't stop this from making its way into the examples 😄

File: github_issue_summarization/pachyderm_seldon/README.md [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+We need to "install" ksonnet packages for the KubeFlow components we will be using. That requires pulling some files from GitHub. To avoid getting rate limited during the tutorial, you will need to export your GitHub API token to an environment variable. Create a new token or use an existing one from [here](https://github.com/settings/tokens), then run:
+
+```
+$ export GITHUB_TOKEN=<your-token-from-gh>
+```
+
+It's recommended to install the core Kubeflow infrastructure, which includes the ability to train models with a TFJob CRD. In addition to that, we are going to go ahead and add in the Pachyderm and Seldon components:
+
+```
+$ ks registry add kubeflow github.com/katacoda/kubeflow-ksonnet/tree/master/kubeflow
```
[the remark this answers is not in the document]
Comment (Nick-Harvey) [reply github_issue_summarization/pachyderm_seldon/README.md]: Fixed

**Comment by Nick-Harvey** [comment]
went ahead and created a github issue on our side so we can make sure to follow up on that promise of a TFjob fulfillment 😄

**Comment by jlewi** [comment]
Can you explain why this PR is so large? 294 files? That makes it pretty difficult to review. It looks like some of this is vendor directories. If you could update the PR description to provide some guidance on what changes this PR is making.

I'd like kubeflow/examples#265 to be submitted first. That PR gets rid of a lot of existing code duplication in the sample.

**Comment by jlewi** [comment]
It looks like this PR is renaming a bunch of files; e.g. 
Renaming/moving files into the directory jupyter_seldon.

Is that correct? 

Do we need to do that? Large scale moves are likely to create a messy tangle of merge conflict problems.

**Comment by jlewi** [comment]
#265 has been merged. So you should now be able to reuse that code in this example.
Sorry for the churn.

**Comment by dsdinter** [comment]
Thanks @Nick-Harvey , it should be in kubeflow/examples/gh_issue_summarization as per @jlewi @texasmichelle comments in previous PR.
I believe the main point now, as per our conversation in Slack, is to whether we need to refactor all existing content in a different subfolder or not:
**kubeflow/examples/github_issue_summarization/jupyter_seldon/<existing content>**
kubeflow/examples/github_issue_summarization/pachyderm_seldon/<pachyderm content>

One approach to avoid the refactoring might be to keep existing content where it is and document that as the "base" example, then describe Pachyderm as an evolution of the base example, @jlewi @texasmichelle what do you think? 
I don't want to overcomplicate things but unless everybody is happy with refactoring the existing code in a new subfolder we need to agree best approach here.
@Nick-Harvey I would suggest to keep the current content intact and focus on kubeflow/examples/github_issue_summarization/pachyderm_seldon/<pachyderm content>
while we all agree about where to put existing content.

I guess everybody's focus is now on Kubecon so we can revisit that after the event, hope you enjoy it!

**Comment by dsdinter** [comment]
@Nick-Harvey I think you need to rebase, if you keep the current content intact will simplify the rebase (Until we agree how to keep the different iterations of this example).

**Comment by texasmichelle** [comment]
Keeping existing files as the "base" example, with everything else in a `pachyderm_seldon` subfolder makes sense. That should clean things up significantly after a rebase.

**Comment by Nick-Harvey** [comment]
Pinged in Slack a couple days ago to try and get this moving again. Any chance I can get a review

**Comment by jlewi** [comment]
@Nick-Harvey @texasmichelle is aware of this but she's a bit backlogged so she might not get it until next week.

**Comment by Nick-Harvey** [comment]
I figured as much :), just wanted to check and see. Thank you in advance @texasmichelle and @jlewi

**Comment by jlewi** [comment]
@Nick-Harvey I wanted to circle back and see how I could help push this along since I know you've been waiting a while.

Looking back at @dsdinter's comment
https://github.com/kubeflow/examples/pulls#issuecomment-445531455

I think the only change needed was to put all the pachyderm code in 
kubeflow/examples/github_issue_summarization/pachyderm_seldon/

Is that done? 

It looks like there a bunch of files in pachyderm_seldon but it also looks like there is a massive number of other files that got changed. I'm guessing that's because it just got out of date and might need to be refreshed.

It looks like in your commit history you have commits from multiple authors; which leads me to believe you might need to rebase and squash commits (did you do a merge the last time you updated)?

Can you sync the code on your branch please.

**Comment by texasmichelle** [comment]
Looks good to go after a rebase

**Comment by jlewi** [comment]
@Nick-Harvey I know this has been a pain; any plans on updating this and trying to get it merged finally?

**Comment by texasmichelle** [comment]
👍 we'd love to see this merged. Let us know if we can help @Nick-Harvey

**Comment by Nick-Harvey** [comment]
Should be good to go now

**Comment by jlewi** [comment]
The CLA check isn't passing because you have commits from multiple authors. If you rebase and squash those commits that should fix it.

Can you revert all changes to files in xgboost_ames_housing please?

**Comment by Nick-Harvey** [comment]
I've attempted a few squashs and rebases at this point, and I've just ended up with a tangled mess of a repo 😄. I'm sure I could figure it out and get it all sorted given some time, but I'm a bit swamped so, I took the easy way out 🥂. Here's a fresh PR with the same content: https://github.com/kubeflow/examples/pull/522

**Pull request closed without merging**